packages/core/helper-plugin/src/hooks/tests/useQueryParams.test.ts

```diff
@@ -0,0 +1,28 @@
+import { renderHook, act } from '@testing-library/react';
+
+import { useQueryParams } from '../useQueryParams';
+
+const mockHistoryPush = jest.fn();
+
+jest.mock('react-router-dom', () => ({
+  ...jest.requireActual('react-router-dom'),
+  useLocation: jest.fn(() => ({
+    search: '',
+  })),
+  useHistory: jest.fn(() => ({
+    push: mockHistoryPush,
+  })),
+}));
```

Comment:
How come we're mocking this instead of using `MemoryRouter`? 🤔  is it because you want to know if the URLs have been updated?

Reply:
With memory router I was getting error for location & history undefined. 
I updated with Router and history prop as mentioned here as alternatives, https://v5.reactrouter.com/web/guides/testing/checking-location-in-tests.